libxrdp/xrdp_caps.c

```diff
@@ -521,6 +522,10 @@ xrdp_caps_process_codecs(struct xrdp_rdp *self, struct stream *s, int len)
     int i1;
     char *codec_guid;
     char *next_guid;
+    struct guid guid;
```

Comment:
I'm not sure `char *next_guid` can be replaced with `struct guid` so just kept.

Reply:
That looks fine to me.